lib/graphql_rails/model/configuration.rb

```diff
@@ -27,16 +27,9 @@ def initialize_copy(other)
       def attribute(attribute_name, **attribute_options)
         key = attribute_name.to_s
 
-        attributes[key] ||= Attributes::Attribute.new(attribute_name)
-
-        attributes[key].tap do |attribute|
```

Comment:
logic moved to https://github.com/samesystem/graphql_rails/pull/211/files#diff-eeb4f2e1d298fe120743ae3e31ca58dfbdd2e9579150dba0941ca1bfbb510b0cR39